File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = ''
    s = s.split(" ")
    
    for item in s:
        tmp = ''
        for i in range(len(item)):
            if i % 2 == 0:
                tmp += item[i].upper()
            else:
                tmp += item[i].lower()
        answer += tmp + " "
    
    return answer[0:len(answer) - 1]

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    s = s.split(" ");
    const answer = [];
    s.forEach((item) => {
        let temp = "";
        for (let i = 0; i < item.length; i++) {
            if (i % 2 === 0) {
                temp += item[i].toUpperCase();
            } else {
                temp += item[i].toLowerCase();
            }
        }
        answer.push(temp);
    })
    return answer.join(" ");
}